README.md

```diff
@@ -1,11 +1,10 @@
-[![Build Status](https://jenkins.napps.dev.nuxeo.com/buildStatus/icon?job=nuxeo%2Fnuxeo-coldstorage%2Flts-2021)](https://jenkins.napps.dev.nuxeo.com/job/nuxeo/job/nuxeo-coldstorage/job/lts-2021/)
-[![Preview Link](https://img.shields.io/badge/preview-available-blue)](https://preview-nuxeo-coldstorage-lts-2021.napps.dev.nuxeo.com/)
+[![Build Status](https://jenkins.platform.dev.nuxeo.com/buildStatus/icon?job=coldstorage%2Fnuxeo-coldstorage%2Flts-2021)](https://jenkins.platform.dev.nuxeo.com/job/coldstorage/job/nuxeo-coldstorage/job/lts-2021/)
```

Comment:
still some occurrences of napps in the file

Reply:
I didn't replaced the occurence in the Crowdin section as it would probably be reworked with https://github.com/nuxeo/nuxeo-coldstorage/pull/226?